README.md

```diff
@@ -64,7 +64,15 @@ module.exports = {
 
 ### Loading Scoped Styles
 
-You need to specify scoped flag and loaders for style files such as `style-loader` and `css-loader`. Note that they must be `enforce: post` to inject scope id into styles before they are processed by them.
+For an explanation of scoped styles, see the [vue-loader docs](https://vue-loader.vuejs.org/en/features/scoped-css.html). 
+
+Html and style files need to be imported using the loader syntax: `require("./file.html?style=./file.css")`. You also need modify your webpack config as follows:
```

Comment:
> 1. Isn't import withRender from './app.html?style=./app.css' a default import? 

Actually, webpack v2, rollup and babel treat the commonjs export (`module.exports`) as the default export (`export default`). You can confirm it if you set `--module es2015` of your typescript compiler options.

> 2. It looks like there's an annoying limitation with the typescript declare module '*.html' 

I think it's worth mentioning in docs how we handle it in TypeScript but I also think it would be better it's as a dedicated section for describing TypeScript setup as a supplement. Because this loader is not only for TypeScript project.